simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt

```diff
@@ -303,8 +317,8 @@ class LocationFragment : Fragment(), LocationListener {
    * Enables the next button and update header status
    */
   private fun enableNext(enable: Boolean) {
-    isNewYork = enable || locationMock
-    binding.nextBtn.isEnabled = enable || locationMock
+    isNewYork = enable || locationMock != null
+    binding.nextBtn.isEnabled = enable || locationMock != null
```

Comment:
I'm wondering - why always enable the button when the locationMock is present? Seems odd.

It doesn't really matter too much since the locationMock won't be set in production code and as you mentioned this component will soon be gone in favor of web-based card creation. I just thought it's a weird bit of logic - like if I call enableNext(false) due to some error state but I'm using a locationMock the app will allow me to move to the next step.

Reply:
We have to enable the next button in order to be able to go to the next screen. Otherwise, the app will be waiting for a real location in New York to enable the next button.